Lab2b/Lab2b/Lab2b.cpp

```diff
@@ -0,0 +1,67 @@
+// Lab2b.cpp : Defines the entry point for the console application.
+//
+
+#include <mpi.h>
+#include <iostream>
+#include <vector>
+#define MASTER 0
+
+int numProcs;
+
+void verifyLab2b(const std::vector<int> input, int nrToSearch, std::vector<int> expectedPos)
+{
+    int procId;
+    MPI_Comm_rank(MPI_COMM_WORLD, &procId);
+    std::vector<int> localInput;
+
+    if (procId == MASTER) 
+    {
+        localInput = input;
+    }
+    else 
+    {
+        localInput.resize(input.size());
```

Comment:
This is a scatter. You don't need the full size of the original vector. Just the "scatter" size